layouts/_default/article.html

```diff
@@ -1,16 +1,19 @@
 <div class="article ">
-    <img src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl" style="height:212px; width:100%">
+    <a class="block" href="{{ .Permalink }}">
+        <img src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl" style="height:212px; width:100%"></a>
```

Comment:
```suggestion
        <img src="{{ .Params.image | absURL }}" alt="{{.alt}}" class="rounded-t-xl" style="height:212px; width:100%"></a>
```

Reply:
The alt text is pulled in like this in the opensource file. Should it be like that here as well?